src/components/Upload.vue

```diff
@@ -1,62 +1,157 @@
 <template >
```

Comment:
Based on file path, I cannot determine what `Upload` is this meant for ? Put it in proper folder !

Reply:
Sir there is already upload.vue as a component in the case and may be used elsewhere !

Reply:
`proof` folder

Reply:
Done Sir